### DISCUSSION TITLE
Update Attribute

### DISCUSSION THREAD

**Discussion by RZ-Jyothsna** [mirumee/saleor-dashboard#3592] [Q&A]
I want to enable 'use in Faceted navigation' for an attribute which already created. It is showing error saying field 'Type' is required. 'Type' value is shown as null in query response. I am using 2.11.1 branch of saleor-dashboard.
![image](https://user-images.githubusercontent.com/117725492/236149873-09a7843c-1214-49f0-b8c9-15a16f685259.png)

**Comment by timuric** [comment]
Hi @RZ-Jyothsna unfortunately we no longer support saleor 2.x
